src/Physics/Plugins/babylon.cannonJSPlugin.ts

```diff
@@ -116,7 +116,7 @@
                                 this.removePhysicsBody(childImpostor);
                                 childImpostor.physicsBody = null;
                             }
-                            childImpostor.parent = mainImpostor;
+                            (childImpostor as any).parent = mainImpostor;

```

Comment:
This is not the way to fix it then :) I don;t want to do a fast fix
Ping @RaananW